source/common/common/to_lower_table.cc

```diff
@@ -0,0 +1,16 @@
+#include "common/common/to_lower_table.h"
+
+ToLowerTable::ToLowerTable() {
```

Comment:
Yeah, I wouldn't worry about it for this since the table is simple and only calculated once at startup.  But, in any case where you have a frequently executed initialization routine that can be done purely at compile time then constexpr is really nice.